srcpkgs/fstrcmp/template

```diff
@@ -0,0 +1,51 @@
+# Template file for 'fstrcmp'
+pkgname=fstrcmp
+version=0.7.D001
+revision=1
+build_style=gnu-configure
+configure_args="LIBTOOL=${XBPS_CROSS_BASE}/usr/bin/libtool"
+make_build_args="LIBTOOL=${XBPS_CROSS_BASE}/usr/bin/libtool"
+make_install_args="LIBTOOL=${XBPS_CROSS_BASE}/usr/bin/libtool"
+hostmakedepends="automake groff ghostscript"
+makedepends="libtool"
+short_desc="Fuzzy comparison of strings"
+maintainer="Elie ROUDNINSKI <xademax@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="http://${pkgname}.sourceforge.net/"
+distfiles="http://${pkgname}.sourceforge.net/${pkgname}-${version}.tar.gz"
+checksum=e4018e850f80700acee8da296e56e15b1eef711ab15157e542e7d7e1237c3476
+
+pre_configure() {
+	rm configure
```

Comment:
why remove the pre-provided configure script ?

Also no need to remove it as autoreconf will overwrite it

Reply:
Just a habit i took because it sometime usufull to regenarate configure / Makefile with uptodate autotools. I will remove the pre_configure step.